integration-tests/tests/aws/resources/dynamodb-helper.ts

```diff
@@ -2,30 +2,65 @@ import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
 import {
   DeleteCommand,
   DynamoDBDocumentClient,
+  DynamoDBDocumentClientResolvedConfig,
   PutCommand,
+  ServiceInputTypes,
+  ServiceOutputTypes,
 } from "@aws-sdk/lib-dynamodb";
+import { Command } from "@aws-sdk/types";
+
+type DynamoDBDocumentClientCommand<
```

Comment:
Should we maybe use `DynamoDBCommand` directly from `@aws-sdk/types` to simplify this definition? If this would break something, then ignore this comment.